lib/sorcery/controller/submodules/remember_me.rb

```diff
@@ -71,7 +71,8 @@ def set_remember_me_cookie!(user)
               value: user.send(user.sorcery_config.remember_me_token_attribute_name),
               expires: user.send(user.sorcery_config.remember_me_token_expires_at_attribute_name),
               httponly: Config.remember_me_httponly,
-              domain: Config.cookie_domain
+              domain: Config.cookie_domain,
+              secure: Config.remember_me_secure
```

Comment:
I'd be worried that this has the potential to overwrite the default of `secure: true` when using `force_ssl`. We'll need to double check that interaction before merging this in. Ideally, adding a test or two.